tsconfig.json

```diff
@@ -16,6 +16,6 @@
     "resolveJsonModule": true,
     "typeRoots": ["./node_modules/@types", "./@types"]
   },
-  "include": ["@types/**/*.ts", "src/**/*.js", "src/**/*.ts",  "src/**/*.d.ts", "src/**/*.tsx"],
+  "include": ["@types/**/*.ts", "src/**/*.ts",  "src/**/*.d.ts", "src/**/*.tsx", "/*.ts"],
```

Comment:
you shouldn't need the `/*.ts` and in fact that may be somewhat dangerous as it may pull in stuff from outside the `rootDir`. The `src/**/*.ts` glob will pull in `src/index.ts`

Reply:
removed and still compiles in client